### DISCUSSION TITLE
Help Sendgrid SMTP Settings

### DISCUSSION THREAD

**Discussion by jamesallchin** [supabase/supabase#3869] [Questions] [answered]
Hi, 

I'm trying to use a custom SMTP setup for sending emails via SendGrid. 

![image](https://user-images.githubusercontent.com/6538573/141351814-42ae57ec-b123-41fb-985c-5652ab2c45c9.png)

I have 💯 confirmed that I can send emails via the SendGrid SMTP gateway use the same credentials (using raw telnet commands). 

When I now initiate Go Auth emails e.g. perform a Sign Up or Magic Link via the API or use the Supabase Dashboard I end with a 500 error response:

`{"error":{"code":500,"message":"Error sending magic link","requestId":"eaaa5c20-0689-45eb-8ce6-4fb191893537"}}
`

Has anyone had any luck with using SendGrid?

I've even tried setting the user and password to base64 encoded strings (this would be required over telnet) but no joy. 

Thanks!

**Comment by jamesallchin** [comment, marked as answer]
Sorry this is my fault - I was being a plonker. The port was incorrect. 

I had entered the telnet port (25) but should have specced the SMTP port of 465.

**Comment by ahmedabdelhafez** [comment]
@jamesallchin can you share the full steps
and please refre that you are using the free plane or the paid method
thanks
